ui/data.js

```diff
@@ -23,7 +23,7 @@
 		factory( jQuery );
 	}
 } ( function( $ ) {
-return $.extend( $.expr[ ":" ], {
+return $.extend( jQuery.expr.pseudos, {
```

Comment:
This has to stay as `$` instead of `jQuery` for `noConflict()` support. `$` is a local reference via the parameter.

I'll make the change when I land this.